RELEASE_NOTES.md

```diff
@@ -2,4 +2,4 @@
 
 ## Summary
 
-* `frequenz-sdk` dependency has been extended to include version `1.0.0-rc1000`.
+This is a hot fix for recurrence not working
```

Comment:
Can you describe better what the issue exactly was here too? As a user I have no idea how my code was affected before this release and what should I expect from this one.

Reply:
There is really only one user and I talked to him already ;)

Reply:
You can assume that it wasn't working at all before, the chance that it was is very low.
If it was working for the user before, they must already have a detailed understanding of this problem or they couldn't have worked around it

Reply:
Well, you don't know for sure, this is a public repo. I'm sure we have plenty of external users :trollface: 

But anyway, I think it is good practice to write notes that are useful, otherwise there isn't much point in writing them at all.

Reply:
Well, I see all the logs and request in the API, I know for sure :P

But.. yes, I suppose it is good practice.